### PR TITLE
narrowed spherical definition

### PR DESCRIPTION
**Pull request by cameronmore**
Addresses #312

### CODE REVIEW

File: QualityOntology.ttl [hunk 1]
```diff
@@ -736,7 +736,7 @@ cco:SpatialOrientation rdf:type owl:Class ;
 ###  http://www.ontologyrepository.com/CommonCoreOntologies/Spherical
 cco:Spherical rdf:type owl:Class ;
               rdfs:subClassOf cco:ThreeDimensionalShape ;
-              cco:definition "A Three Dimensional Shape inhering in a bearer in virtue of it having round cross sections."@en ;
+              cco:definition "A Three Dimensional Shape inhering in a bearer in virtue of it having only round cross sections."@en ;
```
Comment (neilotte) [inline QualityOntology.ttl:739]: @cameronmore It appears @gregfowlerphd offered a further candidate paraphrase on the issue. Please see what you think.

**Review by neilotte — CHANGES_REQUESTED** [review]

File: QualityOntology.ttl (hunk 1 above)
Comment (cameronmore) [reply QualityOntology.ttl]: adjusted

**Comment by mark-jensen** [comment]
@cameronmore 78 commits behind `develop`, please fix conflicts.

**Review by mark-jensen — APPROVED** [review]

**Review by mark-jensen — APPROVED** [review]

**Comment by mark-jensen** [comment]
@cameronmore Hold any attempt to fix until we sort this out across the repo since all PRs are showing conflicts.

**Pull request closed without merging**